06-02 18:57:41 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "PLAY_EVOLUTIONS" не найдена
Table "PLAY_EVOLUTIONS" not found; SQL statement:
select id, hash, apply_script, revert_script, state, last_problem from play_evolutions where state like 'applying_%' [42102-175]
06-25 15:44:08 jdbc[6]: exception
org.h2.jdbc.JdbcSQLException: База данных уже закрыта (чтобы отключить автоматическое закрытие базы данных при останове JVM, добавьте ";DB_CLOSE_ON_EXIT=FALSE" в URL)
Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-175]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:332)
	at org.h2.message.DbException.get(DbException.java:172)
	at org.h2.message.DbException.get(DbException.java:149)
	at org.h2.message.DbException.get(DbException.java:138)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1413)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1388)
	at org.h2.jdbc.JdbcConnection.getMetaData(JdbcConnection.java:309)
	at com.jolbox.bonecp.ConnectionHandle.getMetaData(ConnectionHandle.java:814)
	at com.jolbox.bonecp.BoneCP.isConnectionHandleAlive(BoneCP.java:713)
	at com.jolbox.bonecp.ConnectionTesterThread.run(ConnectionTesterThread.java:103)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
06-26 16:28:25 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "PLAY_EVOLUTIONS" не найдена
Table "PLAY_EVOLUTIONS" not found; SQL statement:
select id, hash, apply_script, revert_script, state, last_problem from play_evolutions where state like 'applying_%' [42102-175]
12-11 00:55:46 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Таблица "LESSON" не найдена
Table "LESSON" not found; SQL statement:
select t0.group_number c0, t0.day c1, t0.hours c2, t0.lecture c3, t0.teacher c4, t0.room c5 from lesson t0 where lower(t0.group_number) like ?  and lower(t0.day) like ?  and lower(t0.hours) like ?  and lower(t0.lecture) like ?  and lower(t0.teacher) like ?  and lower(t0.room) like ?  [42102-175]
12-11 17:01:01 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Таблица "LESSON" не найдена
Table "LESSON" not found; SQL statement:
insert into lesson (group_number, day, hours, lecture, teacher, room) values (?,?,?,?,?,?) [42102-175]
12-11 17:06:53 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Таблица "LESSON" не найдена
Table "LESSON" not found; SQL statement:
insert into lesson (group_number, day, hours, lecture, teacher, room) values (?,?,?,?,?,?) [42102-175]
12-28 22:25:55 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL " 
 +[*] CREATE TABLE LESSON (
  ID                        INTEGER NOT NULL,
  GROUP_NUMBER              VARCHAR(255),
  DAY                       VARCHAR(255),
  HOURS                     VARCHAR(255),
  LECTURE                   VARCHAR(255),
  TEACHER                   VARCHAR(255),
  ROOM                      VARCHAR(255),
  CONSTRAINT PK_LESSON PRIMARY KEY (ID))
; "
Syntax error in SQL statement " 
 +[*] CREATE TABLE LESSON (
  ID                        INTEGER NOT NULL,
  GROUP_NUMBER              VARCHAR(255),
  DAY                       VARCHAR(255),
  HOURS                     VARCHAR(255),
  LECTURE                   VARCHAR(255),
  TEACHER                   VARCHAR(255),
  ROOM                      VARCHAR(255),
  CONSTRAINT PK_LESSON PRIMARY KEY (ID))
; "; SQL statement:
 
 + create table lesson (
  id                        integer not null,
  group_number              varchar(255),
  day                       varchar(255),
  hours                     varchar(255),
  lecture                   varchar(255),
  teacher                   varchar(255),
  room                      varchar(255),
  constraint pk_lesson primary key (id))
; [42000-175]
12-28 22:25:55 jdbc[6]: exception
org.h2.jdbc.JdbcSQLException: Таблица "LESSON" не найдена
Table "LESSON" not found; SQL statement:
select t0.id c0, t0.group_number c1, t0.day c2, t0.hours c3, t0.lecture c4, t0.teacher c5, t0.room c6 from lesson t0 [42102-175]
12-28 22:31:46 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Таблица "LESSON" не найдена
Table "LESSON" not found; SQL statement:
select t0.id c0, t0.group_number c1, t0.day c2, t0.hours c3, t0.lecture c4, t0.teacher c5, t0.room c6 from lesson t0 [42102-175]
12-28 22:31:46 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Таблица "LESSON" не найдена
Table "LESSON" not found; SQL statement:
select t0.id c0, t0.group_number c1, t0.day c2, t0.hours c3, t0.lecture c4, t0.teacher c5, t0.room c6 from lesson t0 [42102-175]
12-28 22:35:10 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Таблица "LESSON" не найдена
Table "LESSON" not found; SQL statement:
select t0.id c0, t0.group_number c1, t0.day c2, t0.hours c3, t0.lecture c4, t0.teacher c5, t0.room c6 from lesson t0 [42102-175]
12-28 22:35:10 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Таблица "LESSON" не найдена
Table "LESSON" not found; SQL statement:
select t0.id c0, t0.group_number c1, t0.day c2, t0.hours c3, t0.lecture c4, t0.teacher c5, t0.room c6 from lesson t0 [42102-175]
12-28 22:36:27 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Таблица "LESSON" не найдена
Table "LESSON" not found; SQL statement:
select t0.id c0, t0.group_number c1, t0.day c2, t0.hours c3, t0.lecture c4, t0.teacher c5, t0.room c6 from lesson t0 [42102-175]
12-28 22:36:27 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Таблица "LESSON" не найдена
Table "LESSON" not found; SQL statement:
select t0.id c0, t0.group_number c1, t0.day c2, t0.hours c3, t0.lecture c4, t0.teacher c5, t0.room c6 from lesson t0 [42102-175]
12-28 22:37:27 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Таблица "LESSON" не найдена
Table "LESSON" not found; SQL statement:
select t0.id c0, t0.group_number c1, t0.day c2, t0.hours c3, t0.lecture c4, t0.teacher c5, t0.room c6 from lesson t0 [42102-175]
12-28 22:37:27 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Таблица "LESSON" не найдена
Table "LESSON" not found; SQL statement:
select t0.id c0, t0.group_number c1, t0.day c2, t0.hours c3, t0.lecture c4, t0.teacher c5, t0.room c6 from lesson t0 [42102-175]
12-28 22:40:59 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Таблица "LESSON" не найдена
Table "LESSON" not found; SQL statement:
select t0.id c0, t0.group_number c1, t0.day c2, t0.hours c3, t0.lecture c4, t0.teacher c5, t0.room c6 from lesson t0 [42102-175]
12-28 22:40:59 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Таблица "LESSON" не найдена
Table "LESSON" not found; SQL statement:
select t0.id c0, t0.group_number c1, t0.day c2, t0.hours c3, t0.lecture c4, t0.teacher c5, t0.room c6 from lesson t0 [42102-175]
12-29 04:35:33 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Столбец "LESSON.HOURS" не найден
Column "LESSON.HOURS" not found; SQL statement:
select t0.id c0, t0.group_number c1, t0.day c2, t0.hours c3, t0.lecture c4, t0.teacher c5, t0.room c6 from lesson t0 where lower(t0.group_number) like ?  order by lesson.hours, t0.hours [42122-175]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:332)
	at org.h2.message.DbException.get(DbException.java:172)
	at org.h2.message.DbException.get(DbException.java:149)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:144)
	at org.h2.command.dml.Select.prepare(Select.java:810)
	at org.h2.command.Parser.prepareCommand(Parser.java:240)
	at org.h2.engine.Session.prepareLocal(Session.java:436)
	at org.h2.engine.Session.prepareCommand(Session.java:379)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1138)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:70)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:267)
	at com.jolbox.bonecp.ConnectionHandle.prepareStatement(ConnectionHandle.java:1024)
	at com.avaje.ebeaninternal.server.query.CQuery.prepareBindExecuteQueryWithOption(CQuery.java:391)
	at com.avaje.ebeaninternal.server.query.CQuery.prepareBindExecuteQuery(CQuery.java:368)
	at com.avaje.ebeaninternal.server.query.CQueryEngine.findMany(CQueryEngine.java:165)
	at com.avaje.ebeaninternal.server.query.DefaultOrmQueryEngine.findMany(DefaultOrmQueryEngine.java:77)
	at com.avaje.ebeaninternal.server.core.OrmQueryRequest.findList(OrmQueryRequest.java:263)
	at com.avaje.ebeaninternal.server.core.DefaultServer.findList(DefaultServer.java:1502)
	at com.avaje.ebeaninternal.server.querydefn.DefaultOrmQuery.findList(DefaultOrmQuery.java:890)
	at controllers.Application.foo(Application.java:99)
	at controllers.Application.smartSearch(Application.java:118)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$9$$anonfun$apply$9.apply(routes_routing.scala:186)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$9$$anonfun$apply$9.apply(routes_routing.scala:186)
	at play.core.Router$HandlerInvokerFactory$$anon$4.resultCall(Router.scala:264)
	at play.core.Router$HandlerInvokerFactory$JavaActionInvokerFactory$$anon$15$$anon$1.invocation(Router.scala:255)
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:55)
	at play.GlobalSettings$1.call(GlobalSettings.java:67)
	at play.core.j.JavaAction$$anonfun$11.apply(JavaAction.scala:82)
	at play.core.j.JavaAction$$anonfun$11.apply(JavaAction.scala:82)
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24)
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24)
	at play.core.j.HttpExecutionContext$$anon$2.run(HttpExecutionContext.scala:40)
	at play.api.libs.iteratee.Execution$trampoline$.execute(Execution.scala:46)
	at play.core.j.HttpExecutionContext.execute(HttpExecutionContext.scala:32)
	at scala.concurrent.impl.Future$.apply(Future.scala:31)
	at scala.concurrent.Future$.apply(Future.scala:492)
	at play.core.j.JavaAction$class.apply(JavaAction.scala:82)
	at play.core.Router$HandlerInvokerFactory$JavaActionInvokerFactory$$anon$15$$anon$1.apply(Router.scala:252)
	at play.api.mvc.Action$$anonfun$apply$1$$anonfun$apply$4$$anonfun$apply$5.apply(Action.scala:130)
	at play.api.mvc.Action$$anonfun$apply$1$$anonfun$apply$4$$anonfun$apply$5.apply(Action.scala:130)
	at play.utils.Threads$.withContextClassLoader(Threads.scala:21)
	at play.api.mvc.Action$$anonfun$apply$1$$anonfun$apply$4.apply(Action.scala:129)
	at play.api.mvc.Action$$anonfun$apply$1$$anonfun$apply$4.apply(Action.scala:128)
	at scala.Option.map(Option.scala:145)
	at play.api.mvc.Action$$anonfun$apply$1.apply(Action.scala:128)
	at play.api.mvc.Action$$anonfun$apply$1.apply(Action.scala:121)
	at play.api.libs.iteratee.Iteratee$$anonfun$mapM$1.apply(Iteratee.scala:483)
	at play.api.libs.iteratee.Iteratee$$anonfun$mapM$1.apply(Iteratee.scala:483)
	at play.api.libs.iteratee.Iteratee$$anonfun$flatMapM$1.apply(Iteratee.scala:519)
	at play.api.libs.iteratee.Iteratee$$anonfun$flatMapM$1.apply(Iteratee.scala:519)
	at play.api.libs.iteratee.Iteratee$$anonfun$flatMap$1$$anonfun$apply$14.apply(Iteratee.scala:496)
	at play.api.libs.iteratee.Iteratee$$anonfun$flatMap$1$$anonfun$apply$14.apply(Iteratee.scala:496)
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24)
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24)
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:41)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(AbstractDispatcher.scala:393)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
12-29 20:30:07 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: База данных уже закрыта (чтобы отключить автоматическое закрытие базы данных при останове JVM, добавьте ";DB_CLOSE_ON_EXIT=FALSE" в URL)
Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-175]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:332)
	at org.h2.message.DbException.get(DbException.java:172)
	at org.h2.message.DbException.get(DbException.java:149)
	at org.h2.message.DbException.get(DbException.java:138)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1413)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1388)
	at org.h2.jdbc.JdbcConnection.getMetaData(JdbcConnection.java:309)
	at com.jolbox.bonecp.ConnectionHandle.getMetaData(ConnectionHandle.java:814)
	at com.jolbox.bonecp.BoneCP.isConnectionHandleAlive(BoneCP.java:713)
	at com.jolbox.bonecp.ConnectionTesterThread.run(ConnectionTesterThread.java:103)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
12-29 20:30:07 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: База данных уже закрыта (чтобы отключить автоматическое закрытие базы данных при останове JVM, добавьте ";DB_CLOSE_ON_EXIT=FALSE" в URL)
Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-175]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:332)
	at org.h2.message.DbException.get(DbException.java:172)
	at org.h2.message.DbException.get(DbException.java:149)
	at org.h2.message.DbException.get(DbException.java:138)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1413)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1388)
	at org.h2.jdbc.JdbcConnection.getMetaData(JdbcConnection.java:309)
	at com.jolbox.bonecp.ConnectionHandle.getMetaData(ConnectionHandle.java:814)
	at com.jolbox.bonecp.BoneCP.isConnectionHandleAlive(BoneCP.java:713)
	at com.jolbox.bonecp.ConnectionTesterThread.run(ConnectionTesterThread.java:103)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
12-29 20:30:07 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: База данных уже закрыта (чтобы отключить автоматическое закрытие базы данных при останове JVM, добавьте ";DB_CLOSE_ON_EXIT=FALSE" в URL)
Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-175]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:332)
	at org.h2.message.DbException.get(DbException.java:172)
	at org.h2.message.DbException.get(DbException.java:149)
	at org.h2.message.DbException.get(DbException.java:138)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1413)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1388)
	at org.h2.jdbc.JdbcConnection.getMetaData(JdbcConnection.java:309)
	at com.jolbox.bonecp.ConnectionHandle.getMetaData(ConnectionHandle.java:814)
	at com.jolbox.bonecp.BoneCP.isConnectionHandleAlive(BoneCP.java:713)
	at com.jolbox.bonecp.ConnectionTesterThread.run(ConnectionTesterThread.java:103)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
12-29 21:01:56 jdbc[34]: null
org.h2.message.DbException: Незакрытое приложением соединение уничтожено сборщиком мусора
The connection was not closed by the application and is garbage collected [90018-175]
	at org.h2.message.DbException.get(DbException.java:172)
	at org.h2.message.DbException.get(DbException.java:149)
	at org.h2.message.DbException.get(DbException.java:138)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:171)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:116)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:90)
	at org.h2.Driver.connect(Driver.java:73)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.jolbox.bonecp.BoneCP.obtainRawInternalConnection(BoneCP.java:351)
	at com.jolbox.bonecp.BoneCP.<init>(BoneCP.java:416)
	at com.jolbox.bonecp.BoneCPDataSource.getConnection(BoneCPDataSource.java:120)
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:240)
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:238)
	at scala.collection.immutable.List.map(List.scala:274)
	at play.api.db.BoneCPPlugin.onStart(DB.scala:238)
	at play.api.Play$$anonfun$start$1$$anonfun$apply$mcV$sp$1.apply(Play.scala:91)
	at play.api.Play$$anonfun$start$1$$anonfun$apply$mcV$sp$1.apply(Play.scala:91)
	at scala.collection.immutable.List.foreach(List.scala:383)
	at play.api.Play$$anonfun$start$1.apply$mcV$sp(Play.scala:91)
	at play.api.Play$$anonfun$start$1.apply(Play.scala:91)
	at play.api.Play$$anonfun$start$1.apply(Play.scala:91)
	at play.utils.Threads$.withContextClassLoader(Threads.scala:21)
	at play.api.Play$.start(Play.scala:90)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:157)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:130)
	at scala.Option.map(Option.scala:145)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1.apply(ApplicationProvider.scala:130)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1.apply(ApplicationProvider.scala:128)
	at scala.util.Success.flatMap(Try.scala:230)
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:128)
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120)
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24)
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24)
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
Caused by: org.h2.jdbc.JdbcSQLException: Незакрытое приложением соединение уничтожено сборщиком мусора
The connection was not closed by the application and is garbage collected [90018-175]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:332)
	... 39 more
12-29 21:05:21 jdbc[48]: java.lang.Exception: Open Stack Trace
	at org.h2.util.CloseWatcher.register(CloseWatcher.java:99)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:117)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:90)
	at org.h2.Driver.connect(Driver.java:73)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.jolbox.bonecp.BoneCP.obtainRawInternalConnection(BoneCP.java:363)
	at com.jolbox.bonecp.BoneCP.obtainInternalConnection(BoneCP.java:269)
	at com.jolbox.bonecp.ConnectionHandle.<init>(ConnectionHandle.java:242)
	at com.jolbox.bonecp.BoneCP.<init>(BoneCP.java:469)
	at com.jolbox.bonecp.BoneCPDataSource.getConnection(BoneCPDataSource.java:120)
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:240)
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:238)
	at scala.collection.immutable.List.map(List.scala:274)
	at play.api.db.BoneCPPlugin.onStart(DB.scala:238)
	at play.api.Play$$anonfun$start$1$$anonfun$apply$mcV$sp$1.apply(Play.scala:91)
	at play.api.Play$$anonfun$start$1$$anonfun$apply$mcV$sp$1.apply(Play.scala:91)
	at scala.collection.immutable.List.foreach(List.scala:383)
	at play.api.Play$$anonfun$start$1.apply$mcV$sp(Play.scala:91)
	at play.api.Play$$anonfun$start$1.apply(Play.scala:91)
	at play.api.Play$$anonfun$start$1.apply(Play.scala:91)
	at play.utils.Threads$.withContextClassLoader(Threads.scala:21)
	at play.api.Play$.start(Play.scala:90)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:157)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:130)
	at scala.Option.map(Option.scala:145)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1.apply(ApplicationProvider.scala:130)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1.apply(ApplicationProvider.scala:128)
	at scala.util.Success.flatMap(Try.scala:230)
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:128)
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120)
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24)
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24)
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)

org.h2.message.DbException: Незакрытое приложением соединение уничтожено сборщиком мусора
The connection was not closed by the application and is garbage collected [90018-175]
	at org.h2.message.DbException.get(DbException.java:172)
	at org.h2.message.DbException.get(DbException.java:149)
	at org.h2.message.DbException.get(DbException.java:138)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:171)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:116)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:90)
	at org.h2.Driver.connect(Driver.java:73)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.jolbox.bonecp.BoneCP.obtainRawInternalConnection(BoneCP.java:351)
	at com.jolbox.bonecp.BoneCP.<init>(BoneCP.java:416)
	at com.jolbox.bonecp.BoneCPDataSource.getConnection(BoneCPDataSource.java:120)
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:240)
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:238)
	at scala.collection.immutable.List.map(List.scala:274)
	at play.api.db.BoneCPPlugin.onStart(DB.scala:238)
	at play.api.Play$$anonfun$start$1$$anonfun$apply$mcV$sp$1.apply(Play.scala:91)
	at play.api.Play$$anonfun$start$1$$anonfun$apply$mcV$sp$1.apply(Play.scala:91)
	at scala.collection.immutable.List.foreach(List.scala:383)
	at play.api.Play$$anonfun$start$1.apply$mcV$sp(Play.scala:91)
	at play.api.Play$$anonfun$start$1.apply(Play.scala:91)
	at play.api.Play$$anonfun$start$1.apply(Play.scala:91)
	at play.utils.Threads$.withContextClassLoader(Threads.scala:21)
	at play.api.Play$.start(Play.scala:90)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:157)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:130)
	at scala.Option.map(Option.scala:145)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1.apply(ApplicationProvider.scala:130)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1.apply(ApplicationProvider.scala:128)
	at scala.util.Success.flatMap(Try.scala:230)
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:128)
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120)
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24)
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24)
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
Caused by: org.h2.jdbc.JdbcSQLException: Незакрытое приложением соединение уничтожено сборщиком мусора
The connection was not closed by the application and is garbage collected [90018-175]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:332)
	... 39 more
12-30 00:08:39 jdbc[22]: null
org.h2.message.DbException: Незакрытое приложением соединение уничтожено сборщиком мусора
The connection was not closed by the application and is garbage collected [90018-175]
	at org.h2.message.DbException.get(DbException.java:172)
	at org.h2.message.DbException.get(DbException.java:149)
	at org.h2.message.DbException.get(DbException.java:138)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:171)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:116)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:90)
	at org.h2.Driver.connect(Driver.java:73)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.jolbox.bonecp.BoneCP.obtainRawInternalConnection(BoneCP.java:363)
	at com.jolbox.bonecp.BoneCP.obtainInternalConnection(BoneCP.java:269)
	at com.jolbox.bonecp.ConnectionHandle.<init>(ConnectionHandle.java:242)
	at com.jolbox.bonecp.PoolWatchThread.fillConnections(PoolWatchThread.java:115)
	at com.jolbox.bonecp.PoolWatchThread.run(PoolWatchThread.java:82)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.h2.jdbc.JdbcSQLException: Незакрытое приложением соединение уничтожено сборщиком мусора
The connection was not closed by the application and is garbage collected [90018-175]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:332)
	... 17 more
12-30 00:26:01 jdbc[55]: java.lang.Exception: Open Stack Trace
	at org.h2.util.CloseWatcher.register(CloseWatcher.java:99)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:117)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:90)
	at org.h2.Driver.connect(Driver.java:73)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.jolbox.bonecp.BoneCP.obtainRawInternalConnection(BoneCP.java:363)
	at com.jolbox.bonecp.BoneCP.obtainInternalConnection(BoneCP.java:269)
	at com.jolbox.bonecp.ConnectionHandle.<init>(ConnectionHandle.java:242)
	at com.jolbox.bonecp.PoolWatchThread.fillConnections(PoolWatchThread.java:115)
	at com.jolbox.bonecp.PoolWatchThread.run(PoolWatchThread.java:82)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)

org.h2.message.DbException: Незакрытое приложением соединение уничтожено сборщиком мусора
The connection was not closed by the application and is garbage collected [90018-175]
	at org.h2.message.DbException.get(DbException.java:172)
	at org.h2.message.DbException.get(DbException.java:149)
	at org.h2.message.DbException.get(DbException.java:138)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:171)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:116)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:90)
	at org.h2.Driver.connect(Driver.java:73)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.jolbox.bonecp.BoneCP.obtainRawInternalConnection(BoneCP.java:351)
	at com.jolbox.bonecp.BoneCP.<init>(BoneCP.java:416)
	at com.jolbox.bonecp.BoneCPDataSource.getConnection(BoneCPDataSource.java:120)
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:240)
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:238)
	at scala.collection.immutable.List.map(List.scala:274)
	at play.api.db.BoneCPPlugin.onStart(DB.scala:238)
	at play.api.Play$$anonfun$start$1$$anonfun$apply$mcV$sp$1.apply(Play.scala:91)
	at play.api.Play$$anonfun$start$1$$anonfun$apply$mcV$sp$1.apply(Play.scala:91)
	at scala.collection.immutable.List.foreach(List.scala:383)
	at play.api.Play$$anonfun$start$1.apply$mcV$sp(Play.scala:91)
	at play.api.Play$$anonfun$start$1.apply(Play.scala:91)
	at play.api.Play$$anonfun$start$1.apply(Play.scala:91)
	at play.utils.Threads$.withContextClassLoader(Threads.scala:21)
	at play.api.Play$.start(Play.scala:90)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:157)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:130)
	at scala.Option.map(Option.scala:145)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1.apply(ApplicationProvider.scala:130)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1.apply(ApplicationProvider.scala:128)
	at scala.util.Success.flatMap(Try.scala:230)
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:128)
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120)
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24)
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24)
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
Caused by: org.h2.jdbc.JdbcSQLException: Незакрытое приложением соединение уничтожено сборщиком мусора
The connection was not closed by the application and is garbage collected [90018-175]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:332)
	... 39 more
12-30 00:26:01 jdbc[55]: java.lang.Exception: Open Stack Trace
	at org.h2.util.CloseWatcher.register(CloseWatcher.java:99)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:117)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:90)
	at org.h2.Driver.connect(Driver.java:73)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.jolbox.bonecp.BoneCP.obtainRawInternalConnection(BoneCP.java:363)
	at com.jolbox.bonecp.BoneCP.obtainInternalConnection(BoneCP.java:269)
	at com.jolbox.bonecp.ConnectionHandle.<init>(ConnectionHandle.java:242)
	at com.jolbox.bonecp.BoneCP.<init>(BoneCP.java:469)
	at com.jolbox.bonecp.BoneCPDataSource.getConnection(BoneCPDataSource.java:120)
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:240)
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:238)
	at scala.collection.immutable.List.map(List.scala:274)
	at play.api.db.BoneCPPlugin.onStart(DB.scala:238)
	at play.api.Play$$anonfun$start$1$$anonfun$apply$mcV$sp$1.apply(Play.scala:91)
	at play.api.Play$$anonfun$start$1$$anonfun$apply$mcV$sp$1.apply(Play.scala:91)
	at scala.collection.immutable.List.foreach(List.scala:383)
	at play.api.Play$$anonfun$start$1.apply$mcV$sp(Play.scala:91)
	at play.api.Play$$anonfun$start$1.apply(Play.scala:91)
	at play.api.Play$$anonfun$start$1.apply(Play.scala:91)
	at play.utils.Threads$.withContextClassLoader(Threads.scala:21)
	at play.api.Play$.start(Play.scala:90)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:157)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:130)
	at scala.Option.map(Option.scala:145)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1.apply(ApplicationProvider.scala:130)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1.apply(ApplicationProvider.scala:128)
	at scala.util.Success.flatMap(Try.scala:230)
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:128)
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120)
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24)
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24)
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)

org.h2.message.DbException: Незакрытое приложением соединение уничтожено сборщиком мусора
The connection was not closed by the application and is garbage collected [90018-175]
	at org.h2.message.DbException.get(DbException.java:172)
	at org.h2.message.DbException.get(DbException.java:149)
	at org.h2.message.DbException.get(DbException.java:138)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:171)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:116)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:90)
	at org.h2.Driver.connect(Driver.java:73)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.jolbox.bonecp.BoneCP.obtainRawInternalConnection(BoneCP.java:351)
	at com.jolbox.bonecp.BoneCP.<init>(BoneCP.java:416)
	at com.jolbox.bonecp.BoneCPDataSource.getConnection(BoneCPDataSource.java:120)
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:240)
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:238)
	at scala.collection.immutable.List.map(List.scala:274)
	at play.api.db.BoneCPPlugin.onStart(DB.scala:238)
	at play.api.Play$$anonfun$start$1$$anonfun$apply$mcV$sp$1.apply(Play.scala:91)
	at play.api.Play$$anonfun$start$1$$anonfun$apply$mcV$sp$1.apply(Play.scala:91)
	at scala.collection.immutable.List.foreach(List.scala:383)
	at play.api.Play$$anonfun$start$1.apply$mcV$sp(Play.scala:91)
	at play.api.Play$$anonfun$start$1.apply(Play.scala:91)
	at play.api.Play$$anonfun$start$1.apply(Play.scala:91)
	at play.utils.Threads$.withContextClassLoader(Threads.scala:21)
	at play.api.Play$.start(Play.scala:90)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:157)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:130)
	at scala.Option.map(Option.scala:145)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1.apply(ApplicationProvider.scala:130)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1.apply(ApplicationProvider.scala:128)
	at scala.util.Success.flatMap(Try.scala:230)
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:128)
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120)
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24)
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24)
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
Caused by: org.h2.jdbc.JdbcSQLException: Незакрытое приложением соединение уничтожено сборщиком мусора
The connection was not closed by the application and is garbage collected [90018-175]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:332)
	... 39 more
12-30 00:26:01 jdbc[55]: null
org.h2.message.DbException: Незакрытое приложением соединение уничтожено сборщиком мусора
The connection was not closed by the application and is garbage collected [90018-175]
	at org.h2.message.DbException.get(DbException.java:172)
	at org.h2.message.DbException.get(DbException.java:149)
	at org.h2.message.DbException.get(DbException.java:138)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:171)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:116)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:90)
	at org.h2.Driver.connect(Driver.java:73)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.jolbox.bonecp.BoneCP.obtainRawInternalConnection(BoneCP.java:351)
	at com.jolbox.bonecp.BoneCP.<init>(BoneCP.java:416)
	at com.jolbox.bonecp.BoneCPDataSource.getConnection(BoneCPDataSource.java:120)
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:240)
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:238)
	at scala.collection.immutable.List.map(List.scala:274)
	at play.api.db.BoneCPPlugin.onStart(DB.scala:238)
	at play.api.Play$$anonfun$start$1$$anonfun$apply$mcV$sp$1.apply(Play.scala:91)
	at play.api.Play$$anonfun$start$1$$anonfun$apply$mcV$sp$1.apply(Play.scala:91)
	at scala.collection.immutable.List.foreach(List.scala:383)
	at play.api.Play$$anonfun$start$1.apply$mcV$sp(Play.scala:91)
	at play.api.Play$$anonfun$start$1.apply(Play.scala:91)
	at play.api.Play$$anonfun$start$1.apply(Play.scala:91)
	at play.utils.Threads$.withContextClassLoader(Threads.scala:21)
	at play.api.Play$.start(Play.scala:90)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:157)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:130)
	at scala.Option.map(Option.scala:145)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1.apply(ApplicationProvider.scala:130)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1.apply(ApplicationProvider.scala:128)
	at scala.util.Success.flatMap(Try.scala:230)
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:128)
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120)
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24)
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24)
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
Caused by: org.h2.jdbc.JdbcSQLException: Незакрытое приложением соединение уничтожено сборщиком мусора
The connection was not closed by the application and is garbage collected [90018-175]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:332)
	... 39 more
12-30 00:35:18 jdbc[62]: java.lang.Exception: Open Stack Trace
	at org.h2.util.CloseWatcher.register(CloseWatcher.java:99)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:117)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:90)
	at org.h2.Driver.connect(Driver.java:73)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.jolbox.bonecp.BoneCP.obtainRawInternalConnection(BoneCP.java:363)
	at com.jolbox.bonecp.BoneCP.obtainInternalConnection(BoneCP.java:269)
	at com.jolbox.bonecp.ConnectionHandle.<init>(ConnectionHandle.java:242)
	at com.jolbox.bonecp.BoneCP.<init>(BoneCP.java:469)
	at com.jolbox.bonecp.BoneCPDataSource.getConnection(BoneCPDataSource.java:120)
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:240)
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:238)
	at scala.collection.immutable.List.map(List.scala:274)
	at play.api.db.BoneCPPlugin.onStart(DB.scala:238)
	at play.api.Play$$anonfun$start$1$$anonfun$apply$mcV$sp$1.apply(Play.scala:91)
	at play.api.Play$$anonfun$start$1$$anonfun$apply$mcV$sp$1.apply(Play.scala:91)
	at scala.collection.immutable.List.foreach(List.scala:383)
	at play.api.Play$$anonfun$start$1.apply$mcV$sp(Play.scala:91)
	at play.api.Play$$anonfun$start$1.apply(Play.scala:91)
	at play.api.Play$$anonfun$start$1.apply(Play.scala:91)
	at play.utils.Threads$.withContextClassLoader(Threads.scala:21)
	at play.api.Play$.start(Play.scala:90)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:157)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:130)
	at scala.Option.map(Option.scala:145)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1.apply(ApplicationProvider.scala:130)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1.apply(ApplicationProvider.scala:128)
	at scala.util.Success.flatMap(Try.scala:230)
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:128)
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120)
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24)
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24)
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)

org.h2.message.DbException: Незакрытое приложением соединение уничтожено сборщиком мусора
The connection was not closed by the application and is garbage collected [90018-175]
	at org.h2.message.DbException.get(DbException.java:172)
	at org.h2.message.DbException.get(DbException.java:149)
	at org.h2.message.DbException.get(DbException.java:138)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:171)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:116)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:90)
	at org.h2.Driver.connect(Driver.java:73)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.jolbox.bonecp.BoneCP.obtainRawInternalConnection(BoneCP.java:351)
	at com.jolbox.bonecp.BoneCP.<init>(BoneCP.java:416)
	at com.jolbox.bonecp.BoneCPDataSource.getConnection(BoneCPDataSource.java:120)
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:240)
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:238)
	at scala.collection.immutable.List.map(List.scala:274)
	at play.api.db.BoneCPPlugin.onStart(DB.scala:238)
	at play.api.Play$$anonfun$start$1$$anonfun$apply$mcV$sp$1.apply(Play.scala:91)
	at play.api.Play$$anonfun$start$1$$anonfun$apply$mcV$sp$1.apply(Play.scala:91)
	at scala.collection.immutable.List.foreach(List.scala:383)
	at play.api.Play$$anonfun$start$1.apply$mcV$sp(Play.scala:91)
	at play.api.Play$$anonfun$start$1.apply(Play.scala:91)
	at play.api.Play$$anonfun$start$1.apply(Play.scala:91)
	at play.utils.Threads$.withContextClassLoader(Threads.scala:21)
	at play.api.Play$.start(Play.scala:90)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:157)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:130)
	at scala.Option.map(Option.scala:145)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1.apply(ApplicationProvider.scala:130)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1.apply(ApplicationProvider.scala:128)
	at scala.util.Success.flatMap(Try.scala:230)
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:128)
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120)
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24)
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24)
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
Caused by: org.h2.jdbc.JdbcSQLException: Незакрытое приложением соединение уничтожено сборщиком мусора
The connection was not closed by the application and is garbage collected [90018-175]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:332)
	... 39 more
12-30 01:09:33 jdbc[15]: java.lang.Exception: Open Stack Trace
	at org.h2.util.CloseWatcher.register(CloseWatcher.java:99)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:117)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:90)
	at org.h2.Driver.connect(Driver.java:73)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.jolbox.bonecp.BoneCP.obtainRawInternalConnection(BoneCP.java:363)
	at com.jolbox.bonecp.BoneCP.obtainInternalConnection(BoneCP.java:269)
	at com.jolbox.bonecp.ConnectionHandle.<init>(ConnectionHandle.java:242)
	at com.jolbox.bonecp.BoneCP.<init>(BoneCP.java:469)
	at com.jolbox.bonecp.BoneCPDataSource.getConnection(BoneCPDataSource.java:120)
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:240)
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:238)
	at scala.collection.immutable.List.map(List.scala:274)
	at play.api.db.BoneCPPlugin.onStart(DB.scala:238)
	at play.api.Play$$anonfun$start$1$$anonfun$apply$mcV$sp$1.apply(Play.scala:91)
	at play.api.Play$$anonfun$start$1$$anonfun$apply$mcV$sp$1.apply(Play.scala:91)
	at scala.collection.immutable.List.foreach(List.scala:383)
	at play.api.Play$$anonfun$start$1.apply$mcV$sp(Play.scala:91)
	at play.api.Play$$anonfun$start$1.apply(Play.scala:91)
	at play.api.Play$$anonfun$start$1.apply(Play.scala:91)
	at play.utils.Threads$.withContextClassLoader(Threads.scala:21)
	at play.api.Play$.start(Play.scala:90)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:157)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:130)
	at scala.Option.map(Option.scala:145)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1.apply(ApplicationProvider.scala:130)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1.apply(ApplicationProvider.scala:128)
	at scala.util.Success.flatMap(Try.scala:230)
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:128)
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120)
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24)
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24)
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)

org.h2.message.DbException: Незакрытое приложением соединение уничтожено сборщиком мусора
The connection was not closed by the application and is garbage collected [90018-175]
	at org.h2.message.DbException.get(DbException.java:172)
	at org.h2.message.DbException.get(DbException.java:149)
	at org.h2.message.DbException.get(DbException.java:138)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:171)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:116)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:90)
	at org.h2.Driver.connect(Driver.java:73)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.jolbox.bonecp.BoneCP.obtainRawInternalConnection(BoneCP.java:351)
	at com.jolbox.bonecp.BoneCP.<init>(BoneCP.java:416)
	at com.jolbox.bonecp.BoneCPDataSource.getConnection(BoneCPDataSource.java:120)
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:240)
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:238)
	at scala.collection.immutable.List.map(List.scala:274)
	at play.api.db.BoneCPPlugin.onStart(DB.scala:238)
	at play.api.Play$$anonfun$start$1$$anonfun$apply$mcV$sp$1.apply(Play.scala:91)
	at play.api.Play$$anonfun$start$1$$anonfun$apply$mcV$sp$1.apply(Play.scala:91)
	at scala.collection.immutable.List.foreach(List.scala:383)
	at play.api.Play$$anonfun$start$1.apply$mcV$sp(Play.scala:91)
	at play.api.Play$$anonfun$start$1.apply(Play.scala:91)
	at play.api.Play$$anonfun$start$1.apply(Play.scala:91)
	at play.utils.Threads$.withContextClassLoader(Threads.scala:21)
	at play.api.Play$.start(Play.scala:90)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:157)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:130)
	at scala.Option.map(Option.scala:145)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1.apply(ApplicationProvider.scala:130)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1.apply(ApplicationProvider.scala:128)
	at scala.util.Success.flatMap(Try.scala:230)
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:128)
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120)
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24)
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24)
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
Caused by: org.h2.jdbc.JdbcSQLException: Незакрытое приложением соединение уничтожено сборщиком мусора
The connection was not closed by the application and is garbage collected [90018-175]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:332)
	... 39 more
12-30 03:01:55 jdbc[65]: java.lang.Exception: Open Stack Trace
	at org.h2.util.CloseWatcher.register(CloseWatcher.java:99)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:117)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:90)
	at org.h2.Driver.connect(Driver.java:73)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.jolbox.bonecp.BoneCP.obtainRawInternalConnection(BoneCP.java:363)
	at com.jolbox.bonecp.BoneCP.obtainInternalConnection(BoneCP.java:269)
	at com.jolbox.bonecp.ConnectionHandle.<init>(ConnectionHandle.java:242)
	at com.jolbox.bonecp.BoneCP.<init>(BoneCP.java:469)
	at com.jolbox.bonecp.BoneCPDataSource.getConnection(BoneCPDataSource.java:120)
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:240)
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:238)
	at scala.collection.immutable.List.map(List.scala:274)
	at play.api.db.BoneCPPlugin.onStart(DB.scala:238)
	at play.api.Play$$anonfun$start$1$$anonfun$apply$mcV$sp$1.apply(Play.scala:91)
	at play.api.Play$$anonfun$start$1$$anonfun$apply$mcV$sp$1.apply(Play.scala:91)
	at scala.collection.immutable.List.foreach(List.scala:383)
	at play.api.Play$$anonfun$start$1.apply$mcV$sp(Play.scala:91)
	at play.api.Play$$anonfun$start$1.apply(Play.scala:91)
	at play.api.Play$$anonfun$start$1.apply(Play.scala:91)
	at play.utils.Threads$.withContextClassLoader(Threads.scala:21)
	at play.api.Play$.start(Play.scala:90)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:157)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:130)
	at scala.Option.map(Option.scala:145)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1.apply(ApplicationProvider.scala:130)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1.apply(ApplicationProvider.scala:128)
	at scala.util.Success.flatMap(Try.scala:230)
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:128)
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120)
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24)
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24)
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)

org.h2.message.DbException: Незакрытое приложением соединение уничтожено сборщиком мусора
The connection was not closed by the application and is garbage collected [90018-175]
	at org.h2.message.DbException.get(DbException.java:172)
	at org.h2.message.DbException.get(DbException.java:149)
	at org.h2.message.DbException.get(DbException.java:138)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:171)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:116)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:90)
	at org.h2.Driver.connect(Driver.java:73)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.jolbox.bonecp.BoneCP.obtainRawInternalConnection(BoneCP.java:351)
	at com.jolbox.bonecp.BoneCP.<init>(BoneCP.java:416)
	at com.jolbox.bonecp.BoneCPDataSource.getConnection(BoneCPDataSource.java:120)
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:240)
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:238)
	at scala.collection.immutable.List.map(List.scala:274)
	at play.api.db.BoneCPPlugin.onStart(DB.scala:238)
	at play.api.Play$$anonfun$start$1$$anonfun$apply$mcV$sp$1.apply(Play.scala:91)
	at play.api.Play$$anonfun$start$1$$anonfun$apply$mcV$sp$1.apply(Play.scala:91)
	at scala.collection.immutable.List.foreach(List.scala:383)
	at play.api.Play$$anonfun$start$1.apply$mcV$sp(Play.scala:91)
	at play.api.Play$$anonfun$start$1.apply(Play.scala:91)
	at play.api.Play$$anonfun$start$1.apply(Play.scala:91)
	at play.utils.Threads$.withContextClassLoader(Threads.scala:21)
	at play.api.Play$.start(Play.scala:90)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:157)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:130)
	at scala.Option.map(Option.scala:145)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1.apply(ApplicationProvider.scala:130)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1.apply(ApplicationProvider.scala:128)
	at scala.util.Success.flatMap(Try.scala:230)
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:128)
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120)
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24)
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24)
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
Caused by: org.h2.jdbc.JdbcSQLException: Незакрытое приложением соединение уничтожено сборщиком мусора
The connection was not closed by the application and is garbage collected [90018-175]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:332)
	... 39 more
12-30 03:01:57 jdbc[72]: java.lang.Exception: Open Stack Trace
	at org.h2.util.CloseWatcher.register(CloseWatcher.java:99)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:117)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:90)
	at org.h2.Driver.connect(Driver.java:73)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.jolbox.bonecp.BoneCP.obtainRawInternalConnection(BoneCP.java:363)
	at com.jolbox.bonecp.BoneCP.obtainInternalConnection(BoneCP.java:269)
	at com.jolbox.bonecp.ConnectionHandle.<init>(ConnectionHandle.java:242)
	at com.jolbox.bonecp.PoolWatchThread.fillConnections(PoolWatchThread.java:115)
	at com.jolbox.bonecp.PoolWatchThread.run(PoolWatchThread.java:85)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)

org.h2.message.DbException: Незакрытое приложением соединение уничтожено сборщиком мусора
The connection was not closed by the application and is garbage collected [90018-175]
	at org.h2.message.DbException.get(DbException.java:172)
	at org.h2.message.DbException.get(DbException.java:149)
	at org.h2.message.DbException.get(DbException.java:138)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:171)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:116)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:90)
	at org.h2.Driver.connect(Driver.java:73)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.jolbox.bonecp.BoneCP.obtainRawInternalConnection(BoneCP.java:351)
	at com.jolbox.bonecp.BoneCP.<init>(BoneCP.java:416)
	at com.jolbox.bonecp.BoneCPDataSource.getConnection(BoneCPDataSource.java:120)
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:240)
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:238)
	at scala.collection.immutable.List.map(List.scala:274)
	at play.api.db.BoneCPPlugin.onStart(DB.scala:238)
	at play.api.Play$$anonfun$start$1$$anonfun$apply$mcV$sp$1.apply(Play.scala:91)
	at play.api.Play$$anonfun$start$1$$anonfun$apply$mcV$sp$1.apply(Play.scala:91)
	at scala.collection.immutable.List.foreach(List.scala:383)
	at play.api.Play$$anonfun$start$1.apply$mcV$sp(Play.scala:91)
	at play.api.Play$$anonfun$start$1.apply(Play.scala:91)
	at play.api.Play$$anonfun$start$1.apply(Play.scala:91)
	at play.utils.Threads$.withContextClassLoader(Threads.scala:21)
	at play.api.Play$.start(Play.scala:90)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:157)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:130)
	at scala.Option.map(Option.scala:145)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1.apply(ApplicationProvider.scala:130)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1.apply(ApplicationProvider.scala:128)
	at scala.util.Success.flatMap(Try.scala:230)
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:128)
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120)
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24)
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24)
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
Caused by: org.h2.jdbc.JdbcSQLException: Незакрытое приложением соединение уничтожено сборщиком мусора
The connection was not closed by the application and is garbage collected [90018-175]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:332)
	... 39 more
12-30 03:27:03 jdbc[29]: null
org.h2.message.DbException: Незакрытое приложением соединение уничтожено сборщиком мусора
The connection was not closed by the application and is garbage collected [90018-175]
	at org.h2.message.DbException.get(DbException.java:172)
	at org.h2.message.DbException.get(DbException.java:149)
	at org.h2.message.DbException.get(DbException.java:138)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:171)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:116)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:90)
	at org.h2.Driver.connect(Driver.java:73)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.jolbox.bonecp.BoneCP.obtainRawInternalConnection(BoneCP.java:351)
	at com.jolbox.bonecp.BoneCP.<init>(BoneCP.java:416)
	at com.jolbox.bonecp.BoneCPDataSource.getConnection(BoneCPDataSource.java:120)
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:240)
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:238)
	at scala.collection.immutable.List.map(List.scala:274)
	at play.api.db.BoneCPPlugin.onStart(DB.scala:238)
	at play.api.Play$$anonfun$start$1$$anonfun$apply$mcV$sp$1.apply(Play.scala:91)
	at play.api.Play$$anonfun$start$1$$anonfun$apply$mcV$sp$1.apply(Play.scala:91)
	at scala.collection.immutable.List.foreach(List.scala:383)
	at play.api.Play$$anonfun$start$1.apply$mcV$sp(Play.scala:91)
	at play.api.Play$$anonfun$start$1.apply(Play.scala:91)
	at play.api.Play$$anonfun$start$1.apply(Play.scala:91)
	at play.utils.Threads$.withContextClassLoader(Threads.scala:21)
	at play.api.Play$.start(Play.scala:90)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:157)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:130)
	at scala.Option.map(Option.scala:145)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1.apply(ApplicationProvider.scala:130)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1.apply(ApplicationProvider.scala:128)
	at scala.util.Success.flatMap(Try.scala:230)
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:128)
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120)
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24)
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24)
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
Caused by: org.h2.jdbc.JdbcSQLException: Незакрытое приложением соединение уничтожено сборщиком мусора
The connection was not closed by the application and is garbage collected [90018-175]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:332)
	... 39 more
12-30 03:27:03 jdbc[29]: null
org.h2.message.DbException: Незакрытое приложением соединение уничтожено сборщиком мусора
The connection was not closed by the application and is garbage collected [90018-175]
	at org.h2.message.DbException.get(DbException.java:172)
	at org.h2.message.DbException.get(DbException.java:149)
	at org.h2.message.DbException.get(DbException.java:138)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:171)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:116)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:90)
	at org.h2.Driver.connect(Driver.java:73)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.jolbox.bonecp.BoneCP.obtainRawInternalConnection(BoneCP.java:351)
	at com.jolbox.bonecp.BoneCP.<init>(BoneCP.java:416)
	at com.jolbox.bonecp.BoneCPDataSource.getConnection(BoneCPDataSource.java:120)
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:240)
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:238)
	at scala.collection.immutable.List.map(List.scala:274)
	at play.api.db.BoneCPPlugin.onStart(DB.scala:238)
	at play.api.Play$$anonfun$start$1$$anonfun$apply$mcV$sp$1.apply(Play.scala:91)
	at play.api.Play$$anonfun$start$1$$anonfun$apply$mcV$sp$1.apply(Play.scala:91)
	at scala.collection.immutable.List.foreach(List.scala:383)
	at play.api.Play$$anonfun$start$1.apply$mcV$sp(Play.scala:91)
	at play.api.Play$$anonfun$start$1.apply(Play.scala:91)
	at play.api.Play$$anonfun$start$1.apply(Play.scala:91)
	at play.utils.Threads$.withContextClassLoader(Threads.scala:21)
	at play.api.Play$.start(Play.scala:90)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:157)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:130)
	at scala.Option.map(Option.scala:145)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1.apply(ApplicationProvider.scala:130)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1.apply(ApplicationProvider.scala:128)
	at scala.util.Success.flatMap(Try.scala:230)
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:128)
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120)
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24)
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24)
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
Caused by: org.h2.jdbc.JdbcSQLException: Незакрытое приложением соединение уничтожено сборщиком мусора
The connection was not closed by the application and is garbage collected [90018-175]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:332)
	... 39 more
12-30 03:27:03 jdbc[29]: null
org.h2.message.DbException: Незакрытое приложением соединение уничтожено сборщиком мусора
The connection was not closed by the application and is garbage collected [90018-175]
	at org.h2.message.DbException.get(DbException.java:172)
	at org.h2.message.DbException.get(DbException.java:149)
	at org.h2.message.DbException.get(DbException.java:138)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:171)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:116)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:90)
	at org.h2.Driver.connect(Driver.java:73)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.jolbox.bonecp.BoneCP.obtainRawInternalConnection(BoneCP.java:351)
	at com.jolbox.bonecp.BoneCP.<init>(BoneCP.java:416)
	at com.jolbox.bonecp.BoneCPDataSource.getConnection(BoneCPDataSource.java:120)
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:240)
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:238)
	at scala.collection.immutable.List.map(List.scala:274)
	at play.api.db.BoneCPPlugin.onStart(DB.scala:238)
	at play.api.Play$$anonfun$start$1$$anonfun$apply$mcV$sp$1.apply(Play.scala:91)
	at play.api.Play$$anonfun$start$1$$anonfun$apply$mcV$sp$1.apply(Play.scala:91)
	at scala.collection.immutable.List.foreach(List.scala:383)
	at play.api.Play$$anonfun$start$1.apply$mcV$sp(Play.scala:91)
	at play.api.Play$$anonfun$start$1.apply(Play.scala:91)
	at play.api.Play$$anonfun$start$1.apply(Play.scala:91)
	at play.utils.Threads$.withContextClassLoader(Threads.scala:21)
	at play.api.Play$.start(Play.scala:90)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:157)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:130)
	at scala.Option.map(Option.scala:145)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1.apply(ApplicationProvider.scala:130)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1.apply(ApplicationProvider.scala:128)
	at scala.util.Success.flatMap(Try.scala:230)
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:128)
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120)
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24)
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24)
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
Caused by: org.h2.jdbc.JdbcSQLException: Незакрытое приложением соединение уничтожено сборщиком мусора
The connection was not closed by the application and is garbage collected [90018-175]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:332)
	... 39 more
12-30 04:06:08 jdbc[67]: java.lang.Exception: Open Stack Trace
	at org.h2.util.CloseWatcher.register(CloseWatcher.java:99)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:117)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:90)
	at org.h2.Driver.connect(Driver.java:73)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.jolbox.bonecp.BoneCP.obtainRawInternalConnection(BoneCP.java:363)
	at com.jolbox.bonecp.BoneCP.obtainInternalConnection(BoneCP.java:269)
	at com.jolbox.bonecp.ConnectionHandle.<init>(ConnectionHandle.java:242)
	at com.jolbox.bonecp.BoneCP.<init>(BoneCP.java:469)
	at com.jolbox.bonecp.BoneCPDataSource.getConnection(BoneCPDataSource.java:120)
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:240)
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:238)
	at scala.collection.immutable.List.map(List.scala:274)
	at play.api.db.BoneCPPlugin.onStart(DB.scala:238)
	at play.api.Play$$anonfun$start$1$$anonfun$apply$mcV$sp$1.apply(Play.scala:91)
	at play.api.Play$$anonfun$start$1$$anonfun$apply$mcV$sp$1.apply(Play.scala:91)
	at scala.collection.immutable.List.foreach(List.scala:383)
	at play.api.Play$$anonfun$start$1.apply$mcV$sp(Play.scala:91)
	at play.api.Play$$anonfun$start$1.apply(Play.scala:91)
	at play.api.Play$$anonfun$start$1.apply(Play.scala:91)
	at play.utils.Threads$.withContextClassLoader(Threads.scala:21)
	at play.api.Play$.start(Play.scala:90)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:157)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:130)
	at scala.Option.map(Option.scala:145)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1.apply(ApplicationProvider.scala:130)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1.apply(ApplicationProvider.scala:128)
	at scala.util.Success.flatMap(Try.scala:230)
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:128)
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120)
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24)
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24)
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)

org.h2.message.DbException: Незакрытое приложением соединение уничтожено сборщиком мусора
The connection was not closed by the application and is garbage collected [90018-175]
	at org.h2.message.DbException.get(DbException.java:172)
	at org.h2.message.DbException.get(DbException.java:149)
	at org.h2.message.DbException.get(DbException.java:138)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:171)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:116)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:90)
	at org.h2.Driver.connect(Driver.java:73)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.jolbox.bonecp.BoneCP.obtainRawInternalConnection(BoneCP.java:363)
	at com.jolbox.bonecp.BoneCP.obtainInternalConnection(BoneCP.java:269)
	at com.jolbox.bonecp.ConnectionHandle.<init>(ConnectionHandle.java:242)
	at com.jolbox.bonecp.PoolWatchThread.fillConnections(PoolWatchThread.java:115)
	at com.jolbox.bonecp.PoolWatchThread.run(PoolWatchThread.java:82)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.h2.jdbc.JdbcSQLException: Незакрытое приложением соединение уничтожено сборщиком мусора
The connection was not closed by the application and is garbage collected [90018-175]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:332)
	... 17 more
12-30 04:06:08 jdbc[67]: java.lang.Exception: Open Stack Trace
	at org.h2.util.CloseWatcher.register(CloseWatcher.java:99)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:117)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:90)
	at org.h2.Driver.connect(Driver.java:73)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.jolbox.bonecp.BoneCP.obtainRawInternalConnection(BoneCP.java:363)
	at com.jolbox.bonecp.BoneCP.obtainInternalConnection(BoneCP.java:269)
	at com.jolbox.bonecp.ConnectionHandle.<init>(ConnectionHandle.java:242)
	at com.jolbox.bonecp.PoolWatchThread.fillConnections(PoolWatchThread.java:115)
	at com.jolbox.bonecp.PoolWatchThread.run(PoolWatchThread.java:85)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)

org.h2.message.DbException: Незакрытое приложением соединение уничтожено сборщиком мусора
The connection was not closed by the application and is garbage collected [90018-175]
	at org.h2.message.DbException.get(DbException.java:172)
	at org.h2.message.DbException.get(DbException.java:149)
	at org.h2.message.DbException.get(DbException.java:138)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:171)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:116)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:90)
	at org.h2.Driver.connect(Driver.java:73)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.jolbox.bonecp.BoneCP.obtainRawInternalConnection(BoneCP.java:363)
	at com.jolbox.bonecp.BoneCP.obtainInternalConnection(BoneCP.java:269)
	at com.jolbox.bonecp.ConnectionHandle.<init>(ConnectionHandle.java:242)
	at com.jolbox.bonecp.PoolWatchThread.fillConnections(PoolWatchThread.java:115)
	at com.jolbox.bonecp.PoolWatchThread.run(PoolWatchThread.java:82)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.h2.jdbc.JdbcSQLException: Незакрытое приложением соединение уничтожено сборщиком мусора
The connection was not closed by the application and is garbage collected [90018-175]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:332)
	... 17 more
12-30 04:06:08 jdbc[67]: null
org.h2.message.DbException: Незакрытое приложением соединение уничтожено сборщиком мусора
The connection was not closed by the application and is garbage collected [90018-175]
	at org.h2.message.DbException.get(DbException.java:172)
	at org.h2.message.DbException.get(DbException.java:149)
	at org.h2.message.DbException.get(DbException.java:138)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:171)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:116)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:90)
	at org.h2.Driver.connect(Driver.java:73)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.jolbox.bonecp.BoneCP.obtainRawInternalConnection(BoneCP.java:363)
	at com.jolbox.bonecp.BoneCP.obtainInternalConnection(BoneCP.java:269)
	at com.jolbox.bonecp.ConnectionHandle.<init>(ConnectionHandle.java:242)
	at com.jolbox.bonecp.PoolWatchThread.fillConnections(PoolWatchThread.java:115)
	at com.jolbox.bonecp.PoolWatchThread.run(PoolWatchThread.java:82)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.h2.jdbc.JdbcSQLException: Незакрытое приложением соединение уничтожено сборщиком мусора
The connection was not closed by the application and is garbage collected [90018-175]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:332)
	... 17 more
12-30 04:46:33 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Недопустимое значение "null" для параметра "SQL"
Invalid value "null" for parameter "SQL" [90008-175]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:332)
	at org.h2.message.DbException.get(DbException.java:172)
	at org.h2.message.DbException.getInvalidValueException(DbException.java:218)
	at org.h2.jdbc.JdbcConnection.translateSQL(JdbcConnection.java:1227)
	at org.h2.jdbc.JdbcConnection.translateSQL(JdbcConnection.java:1214)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:266)
	at com.jolbox.bonecp.ConnectionHandle.prepareStatement(ConnectionHandle.java:1024)
	at com.avaje.ebeaninternal.server.persist.dml.DmlHandler.getPstmt(DmlHandler.java:315)
	at com.avaje.ebeaninternal.server.persist.dml.UpdateHandler.bind(UpdateHandler.java:56)
	at com.avaje.ebeaninternal.server.persist.dml.DmlBeanPersister.execute(DmlBeanPersister.java:79)
	at com.avaje.ebeaninternal.server.persist.dml.DmlBeanPersister.update(DmlBeanPersister.java:66)
	at com.avaje.ebeaninternal.server.persist.DefaultPersistExecute.executeUpdateBean(DefaultPersistExecute.java:82)
	at com.avaje.ebeaninternal.server.core.PersistRequestBean.executeNow(PersistRequestBean.java:452)
	at com.avaje.ebeaninternal.server.core.PersistRequestBean.executeOrQueue(PersistRequestBean.java:478)
	at com.avaje.ebeaninternal.server.persist.DefaultPersister.update(DefaultPersister.java:361)
	at com.avaje.ebeaninternal.server.persist.DefaultPersister.forceUpdateStateless(DefaultPersister.java:234)
	at com.avaje.ebeaninternal.server.persist.DefaultPersister.forceUpdate(DefaultPersister.java:196)
	at com.avaje.ebeaninternal.server.core.DefaultServer.update(DefaultServer.java:1651)
	at com.avaje.ebeaninternal.server.core.DefaultServer.update(DefaultServer.java:1640)
	at com.avaje.ebeaninternal.server.core.DefaultServer.update(DefaultServer.java:1617)
	at com.avaje.ebean.Ebean.update(Ebean.java:496)
	at play.db.ebean.Model.update(Model.java:137)
	at models.ScheduleURL.edit(ScheduleURL.java:29)
	at controllers.Application.editURL(Application.java:152)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$12$$anonfun$apply$12.apply(routes_routing.scala:238)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$12$$anonfun$apply$12.apply(routes_routing.scala:238)
	at play.core.Router$HandlerInvokerFactory$$anon$4.resultCall(Router.scala:264)
	at play.core.Router$HandlerInvokerFactory$JavaActionInvokerFactory$$anon$15$$anon$1.invocation(Router.scala:255)
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:55)
	at play.GlobalSettings$1.call(GlobalSettings.java:67)
	at play.core.j.JavaAction$$anonfun$11.apply(JavaAction.scala:82)
	at play.core.j.JavaAction$$anonfun$11.apply(JavaAction.scala:82)
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24)
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24)
	at play.core.j.HttpExecutionContext$$anon$2.run(HttpExecutionContext.scala:40)
	at play.api.libs.iteratee.Execution$trampoline$.execute(Execution.scala:46)
	at play.core.j.HttpExecutionContext.execute(HttpExecutionContext.scala:32)
	at scala.concurrent.impl.Future$.apply(Future.scala:31)
	at scala.concurrent.Future$.apply(Future.scala:492)
	at play.core.j.JavaAction$class.apply(JavaAction.scala:82)
	at play.core.Router$HandlerInvokerFactory$JavaActionInvokerFactory$$anon$15$$anon$1.apply(Router.scala:252)
	at play.api.mvc.Action$$anonfun$apply$1$$anonfun$apply$4$$anonfun$apply$5.apply(Action.scala:130)
	at play.api.mvc.Action$$anonfun$apply$1$$anonfun$apply$4$$anonfun$apply$5.apply(Action.scala:130)
	at play.utils.Threads$.withContextClassLoader(Threads.scala:21)
	at play.api.mvc.Action$$anonfun$apply$1$$anonfun$apply$4.apply(Action.scala:129)
	at play.api.mvc.Action$$anonfun$apply$1$$anonfun$apply$4.apply(Action.scala:128)
	at scala.Option.map(Option.scala:145)
	at play.api.mvc.Action$$anonfun$apply$1.apply(Action.scala:128)
	at play.api.mvc.Action$$anonfun$apply$1.apply(Action.scala:121)
	at play.api.libs.iteratee.Iteratee$$anonfun$mapM$1.apply(Iteratee.scala:483)
	at play.api.libs.iteratee.Iteratee$$anonfun$mapM$1.apply(Iteratee.scala:483)
	at play.api.libs.iteratee.Iteratee$$anonfun$flatMapM$1.apply(Iteratee.scala:519)
	at play.api.libs.iteratee.Iteratee$$anonfun$flatMapM$1.apply(Iteratee.scala:519)
	at play.api.libs.iteratee.Iteratee$$anonfun$flatMap$1$$anonfun$apply$14.apply(Iteratee.scala:496)
	at play.api.libs.iteratee.Iteratee$$anonfun$flatMap$1$$anonfun$apply$14.apply(Iteratee.scala:496)
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24)
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24)
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:41)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(AbstractDispatcher.scala:393)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
